src/marketplace/eqp/v1/rest-api.md

```diff
@@ -0,0 +1,95 @@
+---
+group: marketplace-api
+title: REST API
+---
+
+ {:.bs-callout-info}
+The **Marketplace EQP REST API** is completely separate from APIs provided with Magento 2.
+<br/>There is no corresponding SOAP API.
+
+The **Marketplace EQP API** gives you access to your **Marketplace Developer Portal** resources.
+These resources are managed by a set of REST API endpoints.
+
+## Overview of resources
+
+| Resource       | Purpose |
+|----------------|---------|
+| authentication | obtain a session token |
+| users          | manage your profile    |
+| files          | manage your profile avatar, ZIP files, image files, PDF documents |
+| packages       | initiate and manage your submissions                        |
+| test results   | receive information about automated and manual reviews      |
+| reports        | get information about your sales, as well as your Magento Marketplace web pages |
+| API callbacks  | request notification as specific workflow activities happen |
+
+## General concepts
+
+### Base URLs
+
+The API endpoints only accept encrypted communications using HTTPS at the following base URLs:
+
+|Environment|Base Url|
+|-----------|--------|
+|sandbox    | `https://developer-stg-api.magento.com` |
+|production | `https://developer-api.magento.com`     |
+
+### REST HTTP verbs
+
+The Marketplace EQP API is based on REST concepts and uses standard HTTP verbs:
+
+|HTTP Verb|Usage|
+|---------|-----|
+| GET     | retrieves a resource, or retrieves a collection of that resource |
```

Comment:
Capitalize the first letter of each entry in the right column.

Reply:
Thank you for making this update!